tests/test_verifyStats.py

```diff
@@ -211,12 +209,10 @@ def setUp(self):
                            'detector': detector.getName(),
                            }
 
-        # Use this to test accessing info from the TaskMetadata:
-        metadataContents = TaskMetadata()
+        # Populate the READ_NOISE into the exposure header
+        md = self.inputExp.getMetadata()
```

Comment:
If you want to be fancy and modern this can be `self.inputExp.metadata`.  But this is also fine.